pkg/doc/verifiable/presentation_jwt_proof_test.go

```diff
@@ -77,29 +79,53 @@ func TestNewPresentationFromJWS(t *testing.T) {
 	})
 }
 
+func TestNewPresentationFromJWS_EdDSA(t *testing.T) {
+	vpBytes := []byte(validPresentation)
+
+	pubKey, privKey, err := ed25519.GenerateKey(rand.Reader)
+	require.NoError(t, err)
+
+	// create presentation
```

Comment:
I think this comment pretty much just restates the line after it (NewPresentation). So I think you can delete this comment

Reply:
Yes, it's almost useless. I'll drop this comment.